### PR TITLE
Sc24 pages

### PR DESCRIPTION
**Pull request by seattleplus**

### CODE REVIEW

File: templates/Pages/cfs_sc24.php [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+        The new ranked lists will be announced at the
+        <?php echo $this->Html->link(__('"IO500: High Performance Storage
+         Community" BoF'),
+            [ 'controller' => 'pages', 'action' => 'display', 'bof-sc24' ],
+            [ 'class' => 'link' ]);
+         ?>.
+         We hope to see many new results.
+    </p>
+    <p>
+        The github repo has a new tag <code>io500-isc24</code> for the current
```
Comment (gflofst) [inline templates/Pages/cfs_sc24.php:29]: Shouldn't this be io500-sc24?

**Review by gflofst — COMMENTED** [review]
I think the one comment with the isc24 tag needs to be updated, but otherwise looks good.

File: templates/Pages/bof_sc24.php [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+    $this->Breadcrumbs->add(__('HOME'), ['controller' => 'submissions', 'action' => 'latest']);
+    $this->Breadcrumbs->add(__('BOF'), ['controller' => 'pages', 'action' => 'display', 'bofs']);
+    $this->Breadcrumbs->add(__("ISC'24"), ['controller' => 'pages', 'action' => 'display', 'bof-isc24']);
+
+    echo $this->Breadcrumbs->render([], ['separator' => ' / ']);
+    ?>
+</nav>
+
+<div class="content">
+    <h2>ISC'24</h2>
```
Comment (adilger) [inline templates/Pages/bof_sc24.php]: Should be SC'24

File: templates/Pages/cfs_sc24.php [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+        submission of small scale results.  View the requirements for submitting
+        to each list on the
+        <?php echo $this->Html->link(__('IO500 Submissions Webpage'),
+            [ 'controller' => 'pages', 'action' => 'display', 'submission' ],
+            [ 'class' => 'link' ]);
+         ?>.
+        The new ranked lists will be announced at the
+        <?php echo $this->Html->link(__('"IO500: High Performance Storage
+         Community" BoF'),
+            [ 'controller' => 'pages', 'action' => 'display', 'bof-isc24' ],
```
Comment (adilger) [inline templates/Pages/cfs_sc24.php]: Schools be bof-sc24

File: templates/Pages/bof_sc24.php [hunk 4]
```diff
@@ -0,0 +1,130 @@
...
+        domain scientists and computer scientists interested in discussing
+        I/O issues.
+        </li>
+    </ul>
+
+    <p>
+        The outcome of this BoF will steer the direction of community efforts.
+    </p>
+
+    <!-- insert presentation link <a class="link" href="/files/ISC24_IO500_Presentation.pdf" target="_blank">BOF Slides</a> -->
```
Comment (adilger) [inline templates/Pages/bof_sc24.php]: Should be SC24

File: templates/Pages/bof_sc24.php [hunk 5]
```diff
@@ -0,0 +1,130 @@
...
+            <tr>
+                <td><strong>Venue</strong></td>
+                <td>Congress Center Hamburg</td>
+                <td>Hall E - 2nd Floor</td>
+            </tr>
+        </tbody>
+    </table>
+
+    <p>
+        Official ISC event link: <a class="link" href="https://app.swapcard.com/widget/event/isc-high-performance-2024/planning/UGxhbm5pbmdfMTgyNjgxNQ==" target="_blank">IO500: The High-Performance Storage Community</a>.
```
Comment (adilger) [inline templates/Pages/bof_sc24.php]: This URL needs to be updated.

File: templates/Pages/bof_sc24.php [hunk 6]
```diff
@@ -0,0 +1,130 @@
...
+        <thread>
+            <th>
+                <td></td>
+                <td></td>
+            </th>
+        </thread>
+        <tbody>
+            <tr>
+                <td><strong>Date</strong></td>
+                <td>Tuesday, May 14, 2024 10:05 AM to 11:05 aM</td>
```
Comment (adilger) [inline templates/Pages/bof_sc24.php]: Date needs to be updated.

File: templates/Pages/bof_sc24.php [hunk 7]
```diff
@@ -0,0 +1,130 @@
+<?php $this->assign('title', 'Birds-of-a-Feather - ISC 24'); ?>
+
+<nav id="breadcrumb">
+    <p>YOU ARE HERE</p>
+
+    <?php
+    $this->Breadcrumbs->add(__('HOME'), ['controller' => 'submissions', 'action' => 'latest']);
+    $this->Breadcrumbs->add(__('BOF'), ['controller' => 'pages', 'action' => 'display', 'bofs']);
+    $this->Breadcrumbs->add(__("ISC'24"), ['controller' => 'pages', 'action' => 'display', 'bof-isc24']);
```
Comment (adilger) [inline templates/Pages/bof_sc24.php]: Schools be SC'24 and bof-sc24.

File: templates/Pages/bof_sc24.php [hunk 8]
```diff
@@ -0,0 +1,130 @@
+<?php $this->assign('title', 'Birds-of-a-Feather - ISC 24'); ?>
```
Comment (adilger) [inline templates/Pages/bof_sc24.php]: Shoukd be SC 24

File: templates/Pages/cfs_sc24.php [hunk 9]
```diff
@@ -0,0 +1,147 @@
+<?php $this->assign('title', 'Call for Submission'); ?>
+
+<div class="content">
+    <h2>Call for Submission</h2>
+    <p class="call">
+        Stabilization Period: April 1st - April 15th, 2024
```
Comment (adilger) [inline templates/Pages/cfs_sc24.php]: Dates need to be updated.

File: templates/Pages/cfs_sc24.php [hunk 10]
```diff
@@ -0,0 +1,147 @@
...
+        <br/>
+        Submission Deadline: May 3rd, 2024 AoE
+    </p>
+
+    <p>
+        The IO500 <strong>is now</strong> accepting and encouraging submissions
+        for the upcoming 14th semi-annual IO500 Production and Research lists,
```
Comment (adilger) [inline templates/Pages/cfs_sc24.php]: This is the 15th list, AFAICS.

File: templates/Pages/cfs_sc24.php [hunk 11]
```diff
@@ -0,0 +1,147 @@
...
+        in conjunction with <strong>ISC'24</strong>.  Once again, we are also
```
Comment (adilger) [inline templates/Pages/cfs_sc24.php]: SC'24

File: templates/Pages/cfs_sc24.php [hunk 12]
```diff
@@ -0,0 +1,147 @@
...
+        Once again, we encourage you to
+        <?php echo $this->Html->link('submit',
+            [ 'controller' => 'pages', 'action' => 'display', 'submission' ],
+            [ 'class' => 'link' ]);
+         ?>
+        to join our community, and to attend the
+        <?php echo $this->Html->link('ISC24 BoF',
+            [ 'controller' => 'pages', 'action' => 'display', 'bof-isc24' ],
+            [ 'class' => 'link' ]);
+         ?>
```
Comment (adilger) [inline templates/Pages/cfs_sc24.php:126]: Fix dates

File: templates/Pages/cfs_sc24.php [hunk 13]
```diff
@@ -0,0 +1,147 @@
...
+
+    <h3>Birds-of-a-Feather</h3>
+    <p>
+        Once again, we encourage you to
+        <?php echo $this->Html->link('submit',
+            [ 'controller' => 'pages', 'action' => 'display', 'submission' ],
+            [ 'class' => 'link' ]);
+         ?>
+        to join our community, and to attend the
+        <?php echo $this->Html->link('ISC24 BoF',
```
Comment (adilger) [inline templates/Pages/cfs_sc24.php]: SC24

File: templates/Pages/cfs_sc24.php [hunk 14]
```diff
@@ -0,0 +1,147 @@
...
+            [ 'controller' => 'pages', 'action' => 'display', 'bof-isc24' ],
```
Comment (adilger) [inline templates/Pages/cfs_sc24.php]: bof-sc24

**Review by adilger — CHANGES_REQUESTED** [review]
Lots of places need to replace "ISC" with "SC"

**Review by adilger — CHANGES_REQUESTED** [review]
It was confusing that the first commit was only a copy of the ISC pages, and the later patches were doing the updates.

It would have been more clear if this was explained in the commit message. 

I _think_ the pages have been fixed, but I didn't see an update to the main BOF/Submissions pages to link to these new pages?

File: templates/Pages/cfs_sc24.php (hunk 1 above)
Comment (seattleplus) [reply templates/Pages/cfs_sc24.php]: yes, udpating

**Comment by seattleplus** [comment]
I updated the bofs and news page (I don't see a place to link in the CFS) and fixed the typo. How does this look?

**Review by adilger — APPROVED** [review]
LGTM

**Comment by gmarkomanolis** [comment]
Approve

**Review by gmarkomanolis — APPROVED** [review]

**Pull request merged**